### ISSUE TITLE
When edit a vtl file from edit mode the vtl button disappears 

### ISSUE THREAD

**Issue by fmontes** [dotCMS/core#19871]
**Describe the bug**
When you edit a vtl file form edit mode and contentlet reload the vtl button doesn't show anymore.

**To Reproduce**
1. Edit a page that have a vtl file in any contentlet
2. Click on the [vtl file button](https://images.zenhubusercontent.com/570fbdcebf93871ccd997222/9fd02cc9-5911-491a-a6a8-38931818f7dd) and then hit save or publish
3. See! the vtl button is gone

**Expected behavior**
It should maintain the button

**Screenshots**
[Screen Shot 2021-01-26 at 3.06.12 PM.png](https://images.zenhubusercontent.com/570fbdcebf93871ccd997222/9fd02cc9-5911-491a-a6a8-38931818f7dd)

**Additional context**
The problem is that the endpoint we use to get the contentlet inside the container is not returning the vtl code

The API url is: `/api/v1/containers/content/CONTENTLET?containerId=CONTAINER_ID`

The HTML looks like this:

```
<div data-dot-object="vtl-file" data-dot-inode="c7fde81d-29bd-439d-9709-06a5a599f347" data-dot-url="storeProductList.vtl" data-dot-can-read="true" data-dot-can-edit="true"></div>
```

**Comment by rjvelazco** [comment]
After editing a vtl file the vtl button disappears because the end point is not returning the vtl button code that it's used to build the button in the FrontEnd. 

API url: `/api/v1/containers/content/CONTENTLET?containerId=CONTAINER_ID`

**Example:**

/api/v1/containers/content/6faf3063-5478-4e0a-a44b-dba540ec79b8?containerId=//default/application/containers/system/

**Current Response:**
```json
{"entity":{"render":"<div data-dot-object=\"contentlet\" data-dot-inode=\"b3f958b4-3705-46e3-9a96-906c5a01cbdf\" data-dot-identifier=\"6faf3063-5478-4e0a-a44b-dba540ec79b8\" data-dot-type=\"FileAsset\" data-dot-basetype=\"FILEASSET\" data-dot-lang=\"1\" data-dot-title=\"Properties.vtl\" data-dot-can-edit=\"true\" data-dot-content-type-id=\"33888b6f-7a8e-4069-b1b6-5c1aa9d0a48d\" data-dot-has-page-lang-version=\"true\"></div>"},"errors":[],"i18nMessagesMap":{},"messages":[],"permissions":[]}
```

**Expected response looks like:**
```
{"entity":{"render":"<div data-dot-object=\"contentlet\" data-dot-inode=\"b3f958b4-3705-46e3-9a96-906c5a01cbdf\" data-dot-identifier=\"6faf3063-5478-4e0a-a44b-dba540ec79b8\" data-dot-type=\"FileAsset\" data-dot-basetype=\"FILEASSET\" data-dot-lang=\"1\" data-dot-title=\"Properties.vtl\" data-dot-can-edit=\"true\" data-dot-content-type-id=\"33888b6f-7a8e-4069-b1b6-5c1aa9d0a48d\" data-dot-has-page-lang-version=\"true\"><div data-dot-object="vtl-file" data-dot-inode="208eb135-3fec-40ce-91be-7ba7914ffd92" data-dot-url="default_container.vtl" data-dot-can-read="true" data-dot-can-edit="true"></div></div>"},"errors":[],"i18nMessagesMap":{},"messages":[],"permissions":[]}
```

In the expected response, inside the contentlet there's a div with the vtl button code.

**Comment by bryanboza** [comment]
Fixed, tested on release-21.10 // docker // FF
